<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serialMode</class>
 <widget class="QDialog" name="serialMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>80</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#serialMode {
border-image: url(:/pic/serial_bak.png);
}</string>
  </property>
  <widget class="QPushButton" name="check_button">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>40</y>
     <width>240</width>
     <height>600</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/check_green_up.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>240</width>
     <height>280</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/logout_up.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="list_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>240</width>
     <height>320</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/list_up.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="time_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>700</y>
     <width>420</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 153, 51);</string>
   </property>
   <property name="text">
    <string>2013-08-07 18:00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="lane_label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>700</y>
     <width>420</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 153, 51);</string>
   </property>
   <property name="text">
    <string>12345</string>
   </property>
  </widget>
  <widget class="QLabel" name="cashier_label">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>700</y>
     <width>420</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 153, 51);</string>
   </property>
   <property name="text">
    <string>     123456</string>
   </property>
  </widget>
  <widget class="QPushButton" name="title_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/zero.png);
color: rgb(255, 153, 51);</string>
   </property>
   <property name="text">
    <string>title</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_E1">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>35</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>E1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_S">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>35</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_I1">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>35</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>I1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_I2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>35</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>I2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_I3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>35</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>I3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_E2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>35</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>E2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_state">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>130</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>state</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_hide">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>640</y>
     <width>420</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/zero.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_error">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>10</y>
     <width>200</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>state</string>
   </property>
  </widget>
  <widget class="QPushButton" name="time_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>640</y>
     <width>420</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/zero.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lane_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>640</y>
     <width>420</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/zero.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cam_label">
   <property name="geometry">
    <rect>
     <x>248</x>
     <y>50</y>
     <width>784</width>
     <height>580</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/standby.png);</string>
   </property>
   <property name="text">
    <string>camera pic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tick_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>237</width>
     <height>187</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/tick.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="merchan_text">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>780</width>
     <height>580</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_EXT">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>35</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>EXT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="recover_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/zero.png);</string>
   </property>
   <property name="text">
    <string>!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
